ib_manifest_util/cli.py

```diff
@@ -31,31 +16,79 @@ def main() -> None:
 
 
 @click.command(
-    "update_manifest",
+    "update_repo",
     help="Update the local hardening manifest and Dockerfile with necessary packages given an environment file",
 )
-@click.option("--file", default=None, help="Path to environment file")
 @click.option(
-    "--startup-scripts-path",
-    default=None,
-    help="Path to .yaml file containing additional files to copy",
+    "--repo_dir",
+    prompt="Please enter the absolute path of the Iron Bank repo to modify",
+)
+@click.option(
+    "--dockerfile_version", prompt="Please enter the desired docker image version"
+)
+@click.option(
+    "--local_env_path",
+    default="local_channel_env.yaml",
+    help="Path to local environment file",
+)
+@click.option(
+    "--startup_scripts_path",
+    help="(Optional) Path to .yaml file containing additional files to copy",
 )
 @click.option(
-    "--dockerfile-version", prompt="Please enter the desired docker image version"
+    "--output_hardening_path",
+    help="(Optional) Path to location in which the hardening manifest will be placed",
 )
-def update_manifest(file, dockerfile_version, startup_scripts_path):
-    # test_existence_copy_markers()
-    create_ib_manifest(file)
-    update_hardening_manifest(dockerfile_version, startup_scripts_path)
-    copy_text = generate_copy_statements(
-        hardening_path=hardening_manifest_path,
-        startup_config_path=startup_scripts_config_path,
+@click.option(
+    "--output_dockerfile_path",
+    help="(Optional) Path to location in which the Dockerfile will be placed",
+)
+def update_repo_cli(
+    repo_dir: str | Path,
+    dockerfile_version: str,
+    local_env_path: str | Path = None,
+    startup_scripts_path: str | Path | None = None,
+    output_hardening_path: str | Path | None = None,
+    output_dockerfile_path: str | Path | None = None,
+):
+    update_repo(
+        repo_dir,
+        dockerfile_version,
+        local_env_path,
+        startup_scripts_path,
+        output_hardening_path,
+        output_dockerfile_path,
```

Comment:
I added an additional parameter here, `output_dockerfile_template`. Its just been merged into dev. Can you update the cli to accept it?

Merging dev into your branch will probably also fix all those merge conflicts.